### PR TITLE
changed to_pandas to toPandas

### PR DESCRIPTION
**Pull request by djemec**
given that retrieve_fields returns a spark dataframe and not a query proxy, to_pandas needs to be "toPandas" so tweaked that markdown

also tweaked the connect to no longer have dialect hard coded since it auto-detects

### CODE REVIEW

File: dxdata/getting_started_with_dxdata.ipynb [hunk 1]
```diff
@@ -393,6 +393,19 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.7.7"
+  },
```
Comment (djemec) [inline dxdata/getting_started_with_dxdata.ipynb:396]: this is some items to help support a table of contents.  doesn't impact usage

**Comment by arunbhat** [comment]
so to_koalas stays? kind of inconsistent.. but @keegankelsey  should review

**Comment by djemec** [comment]
> so to_koalas stays? kind of inconsistent.. but @keegankelsey should review

@arunbhat  yeah, here's koalas function docs https://koalas.readthedocs.io/en/latest/reference/api/databricks.koalas.DataFrame.to_koalas.html

**Comment by djemec** [comment]
> > so to_koalas stays? kind of inconsistent.. but @keegankelsey should review
> 
> @arunbhat yeah, here's koalas function docs https://koalas.readthedocs.io/en/latest/reference/api/databricks.koalas.DataFrame.to_koalas.html

I actually removed koalas from it and used native spark dataframes

**Comment by keegankelsey** [comment]
I would change the initial description from
`...A Dataset object contains content that describes logical content in data and maps the logical data organization to the physical location of data stored in a database. It allows assign something a "Sample ID," a term with specific attributes, and easily store, retrieve, and use that term in a consistent manner. Below, we walk through highlights of dxdata specifically around initiating a spark cluster, pulling a datasets/cohort, exploring the data, and retrieving data to a dataframe`
to
`...A Dataset object maps the logical data model to physical storage, providing the means to retrieve data in a standardized manner. Below, we highlight aspects of dxdata. Specifically we show (a) how initiating a Spark cluster, (b) how to load and access metadata within Datasets and Cohorts, and (c) how to retrieve stored data as specified in Datasets and Cohorts.`

**Comment by keegankelsey** [comment]
I can't access the Dataset, `record-G3814k006Fjgk5J74Kx3X1QF`. Will a public Dataset be provided? If not, access to specific fields used in this notebook may fail if the dataset is not aligned with content in the demo. I.E, not just any Dataset will work to load and run through the script.

This also means I can't fully test the notebook. Can you provide access to the dataset?

**Comment by keegankelsey** [comment]
Same comment regarding access to the demo cohort and this line:

`cohort = dxdata.load_cohort(folder="/", name="controls")`

**Comment by djemec** [comment]
> I would change the initial description from
> `...A Dataset object contains content that describes logical content in data and maps the logical data organization to the physical location of data stored in a database. It allows assign something a "Sample ID," a term with specific attributes, and easily store, retrieve, and use that term in a consistent manner. Below, we walk through highlights of dxdata specifically around initiating a spark cluster, pulling a datasets/cohort, exploring the data, and retrieving data to a dataframe`
> to
> `...A Dataset object maps the logical data model to physical storage, providing the means to retrieve data in a standardized manner. Below, we highlight aspects of dxdata. Specifically we show (a) how initiating a Spark cluster, (b) how to load and access metadata within Datasets and Cohorts, and (c) how to retrieve stored data as specified in Datasets and Cohorts.`

language updated

**Comment by djemec** [comment]
> I can't access the Dataset, `record-G3814k006Fjgk5J74Kx3X1QF`. Will a public Dataset be provided? If not, access to specific fields used in this notebook may fail if the dataset is not aligned with content in the demo. I.E, not just any Dataset will work to load and run through the script.
> 
> This also means I can't fully test the notebook. Can you provide access to the dataset?

We do NOT plan on making the dataset publicly available.  George is making a slightly tweaked version for UKB that people should be able to use, but this is meant to be a starter where they replace the names and ids.
for testing use https://staging.dnanexus.com/panx/projects/G45g7fQ0yxky3kz04JF31bkK/data/

**Comment by djemec** [comment]
> Same comment regarding access to the demo cohort and this line:
> 
> `cohort = dxdata.load_cohort(folder="/", name="controls")`

can use https://staging.dnanexus.com/panx/projects/G45g7fQ0yxky3kz04JF31bkK/data/

**Comment by keegankelsey** [comment]
@djemec - everything looks good except the `to_koalas()` comment. See my notebook screen shots on PySpark and Koalas and use of `toPandas()`, `to_pandas()`, and `to_koalas()`.  Something like this workflow may be good to put in the notebook to avoid future confusion.  There are two  screenshots...just couldn't fit the full comment into a single screen. Message me if you want to chat  through this.
Screen Shot 2021-08-05 at 9 09 58 AM
Screen Shot 2021-08-05 at 9 10 07 AM

**Review by keegankelsey — CHANGES_REQUESTED** [review]
@djemec Please see my comments on `to_koalas()`. This should be updated in your notebook.

**Review by keegankelsey — APPROVED** [review]

**Comment by djemec** [comment]
> @djemec Please see my comments on `to_koalas()`. This should be updated in your notebook.

Just to close out before merging, to_koalas and toPandas were added to the notebook

**Pull request merged**